test/common.hpp

```diff
@@ -51,7 +51,7 @@
     }
 #endif
 
-
+#ifdef ROCWMMA_BENCHMARK_TESTS
 #ifndef CHECK_RSMI_ERROR
 #define CHECK_RSMI_ERROR(status)                            \
     if(status != RSMI_STATUS_SUCCESS )                      \
```

Comment:
Just like prev. adjustments with the CHECK_ERROR_* macros, we need to store the status in a variable so that it gets called only once, otherwise gets called 3 times where 'status' is expanded by the macro